shared/profile/pgp/add.native.js

```diff
@@ -1,8 +1,7 @@
 // @flow
 import React, {Component} from 'react'
-import {Button, Input, PlatformIcon, SmallInput, StandardScreen, Text} from '../../common-adapters'
+import {Button, Input, KeyboardAvoidingBox, PlatformIcon, SmallInput, StandardScreen, Text} from '../../common-adapters/index.native'
```

Comment:
I think putting the `.native` is preffered since you need native only components (keyboard avoid view)